### PR TITLE
feature(nemesis_publisher): report each nemesis result into elastic

### PR DESCRIPTION
**Pull request by fruch**
since we want to track better which nemesis code has been run/failed
using elasticsearch and kiband dashboard.
we are sending each nemesis result, stright away in a flatter data structure

## PR pre-checks (self review)

- [x] I followed [KISS principle](https://en.wikipedia.org/wiki/KISS_principle) and [best practices](https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI)
- [x] I didn't leave commented-out/debugging code
- [x] I added the relevant `backport` labels
- [ ] New configuration option are added and documented (in `sdcm/sct_config.py`)
- [ ] I have added tests to cover my changes (Infrastructure only - under `unit-test/` folder)
- [x] All new and existing unit tests passed (CI)
- [ ] I have updated the Readme/doc folder accordingly (if needed)

### CODE REVIEW

**Comment by fruch** [comment]
taking it for a ride: 
https://jenkins.scylladb.com/job/scylla-staging/job/fruch/job/longevity/job/longevity-100gb-4h-test/7/

**Comment by fruch** [comment]
first job start report into the new index: 
https://70f106c98484448dbc4705050eb3f7e9.us-east-1.aws.found.io:9243/goto/6083ac8db4419fc3adf4b6913a350975

File: sdcm/nemesis_publisher.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+from elasticsearch import Elasticsearch
+
+from sdcm.keystore import KeyStore
+
+LOGGER = logging.getLogger(__name__)
+
+
+class NemesisElasticSearchPublisher:
+    index_name = 'new_nemesis_playground'
```
Comment (fruch) [inline sdcm/nemesis_publisher.py:25]: @roydahan , you can get to name it :)

Comment (fruch) [reply sdcm/nemesis_publisher.py]: until you got to it, I've decide on a name.

**Review by roydahan — APPROVED** [review]
Overall looks good to me.
Do you think we can record somehow the avg_op_rate metric during the nemesis?
I would like to be able to filter out nemesis that run without workload in the background due to issue in the test (or very low workload).

If possible it can be a separate PR.

**Comment by fruch** [comment]
> Overall looks good to me.
> Do you think we can record somehow the avg_op_rate metric during the nemesis?
> I would like to be able to filter out nemesis that run without workload in the background due to issue in the test (or very low workload).
> 
> If possible it can be a separate PR.

We can add anything we want, just need run the needed code and define a proper name for it, it's a bit trick cause we have different metrics for different stress tools.

**Pull request merged**